### PR TITLE
added avoid_stopovers options to directions client

### PR DESCRIPTION
**Pull request by roidnn**
added avoid_stopovers options to directions client so to allow the use of via: prefix and retrieval of duration_in_traffic data

### CODE REVIEW

**Comment by roidnn** [comment]
I signed it!

File: googlemaps/directions.py [hunk 1]
```diff
@@ -24,7 +24,8 @@ def directions(client, origin, destination,
                mode=None, waypoints=None, alternatives=False, avoid=None,
                language=None, units=None, region=None, departure_time=None,
                arrival_time=None, optimize_waypoints=False, transit_mode=None,
-               transit_routing_preference=None, traffic_model=None):
+               transit_routing_preference=None, traffic_model=None,
+               avoid_stopovers=None):
```
Comment (stephenmcd) [inline googlemaps/directions.py]: Should the default be False?

Comment (roidnn) [reply googlemaps/directions.py]: Yes, has more sense (doesn't affect the logic)

**Comment by stephenmcd** [comment]
Thanks for this. Actually I'm not sure it's the best idea thinking about it some more.

The "via:" prefix isn't something that's necessarily applied to all waypoints - it should be applied individually. The developer might have some waypoints already prefixed with "via:" and if they use the proposed arg they'd end up with multiple "via:" prefixes for those waypoints. Given the potential confusion there I don't think we should add this. 

Am I missing anything?

**Comment by roidnn** [comment]
Thank you for addressing this. I understand what you are saying, and it got me checking what happens when the API receives a double "via:" prefix and it looks like a bug, you can try the following:

https://maps.googleapis.com/maps/api/directions/json?departure_time=1529316770&destination=40.76404%2C-73.99225&origin=40.76092%2C-73.99453&waypoints=via:40.76155%2C-73.99407&key=

vs:

https://maps.googleapis.com/maps/api/directions/json?departure_time=1529316770&destination=40.76404%2C-73.99225&origin=40.76092%2C-73.99453&waypoints=via:via:40.76155%2C-73.99407&key=

It adds a weird stop on the way. Maybe worth checking why...

I do think this option is important enough because Google API specifies that in order to get the "duration_in_traffic" data you must obey 5 conditions, one of them is including the "way:" prefix. I believe it quite cumbersome to add it explicitly and one should be able to get this data effortlessly -
 This doesn't seem like an edge-case.

**Pull request closed without merging**